services/src/main/resources/db/changelog/gamification.db.changelog-1.0.0.xml

```diff
@@ -0,0 +1,149 @@
+    <property name="now" value="now()" dbms="h2"/>
+    <property name="now" value="now()" dbms="mysql"/>
+    <property name="autoIncrement" value="true"/>
+
+    <!-- Definition of GAMIFICATION_BADGE table -->
+    <changeSet author="exo-gamification" id="1.0.0-1">
+        <createTable tableName="GAMIFICATION_BADGE">
+            <column name="ID" type="bigint" autoIncrement="${autoIncrement}">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="TITLE" type="NVARCHAR(32)">
+                <constraints nullable="false" unique="true"/>
+            </column>
+            <column name="DESCRIPTION" type="NVARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="NEEDED_SCORE" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="ICON_FILE_ID" type="BIGINT">
+                <constraints nullable="true"/>
+            </column>
+            <column name="VALIDITY_DATE_START" type="TIMESTAMP">
+                <constraints nullable="true"/>
+            </column>
+            <column name="VALIDITY_DATE_END" type="TIMESTAMP">
+                <constraints nullable="true"/>
+            </column>
+            <column name="BADGE_IS_ENABLE" type="BIT" defaultValue="0">
+                <constraints nullable="false"/>
+            </column>
+            <column name="CREATED_BY" type="varchar(70)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="ZONE" type="varchar(70)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="CREATED_DATE" type="timestamp" defaultValueDate="${now}">
+                <constraints nullable="false"/>
+            </column>
+            <column name="LAST_MODIFIED_BY" type="varchar(50)"/>
+            <column name="LAST_MODIFIED_DATE" type="timestamp"/>
+        </createTable>
+        <createTable tableName="GAMIFICATION_RULE">
+            <column name="ID" type="bigint" autoIncrement="${autoIncrement}">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="TITLE" type="NVARCHAR(70)">
+                <constraints nullable="false" unique="true"/>
+            </column>
+            <column name="DESCRIPTION" type="NVARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="SCORE" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="AREA" type="NVARCHAR(32)">
+                <constraints nullable="true"/>
+            </column>
+            <column name="RULE_IS_ENABLE" type="BIT" defaultValue="0">
+                <constraints nullable="false"/>
+            </column>
+            <column name="CREATED_BY" type="varchar(50)">
```

Comment:
the length of CREATED_BY/UPDATED_BY fields is usually 200

Reply:
Done